serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc

```diff
@@ -1 +1,475 @@
-//= Configuring OpenAPI services endpoints in different environments
\ No newline at end of file
+= Configuring OpenAPI services endpoints
+
+:compat-mode!:
+// Metadata:
+:description: Configuring OpenAPI services endpoints in Serverless Workflow
+:keywords: kogito, workflow, quarkus, serverless, openapi, orchestration
+// links:
+:quarkus-profiles-url: https://quarkus.io/guides/config-reference#profiles
+
+This document describes how you can configure OpenAPI service endpoints in Serverless Workflow.
+
+== Configuring OpenAPI services endpoints
+
+Kogito leverages MicroProfile Rest Client to invoke OpenAPI services.
+This means that you can configure the OpenAPI services by following the MicroProfile Config specification.
+
+You can find the list of properties you can configure in the https://download.eclipse.org/microprofile/microprofile-rest-client-2.0/microprofile-rest-client-spec-2.0.html#mpconfig[MicroProfile Rest Client specification].
+
+Kogito generates the Rest Client by following the strategy defined in the `kogito.sw.operationIdStrategy` property.
+This property can have one of the following values:
+
+* `FILE_NAME` - default value.
+Kogito will use a configuration key based on the OpenAPI document file name.
++
+--
+.Example
+Given an OpenAPI document located at `src/main/resources/openapi/stock-portfolio-svc.yaml`, Kogito will use `stock_portfolio_svc_yaml` as configuration key.
+
+[source,properties]
+----
+quarkus.rest-client."stock_portfolio_svc_yaml".url=http://localhost:8282/
```

Comment:
quotes not needed, property name is
quarkus.rest-client.stock_portfolio_svc_yaml.url